device-types/Cisco/ASR1001-HX.yaml

```diff
@@ -0,0 +1,51 @@
+manufacturer: Cisco
+model: ASR1001-HX
+slug: asr1001-hx
+u_height: 1
+is_full_depth: true
+interfaces:
+  - name: GigabitEthernet0/0/0
+    type: 1000base-t
+  - name: GigabitEthernet0/0/1
+    type: 1000base-t
+  - name: GigabitEthernet0/0/2
+    type: 1000base-t
+  - name: GigabitEthernet0/0/3
+    type: 1000base-t
+  - name: GigabitEthernet0/0/4
+    type: 1000base-t
+  - name: GigabitEthernet0/0/5
+    type: 1000base-t
+  - name: GigabitEthernet0/0/6
+    type: 1000base-t
+  - name: GigabitEthernet0/0/7
+    type: 1000base-t
+  - name: Te0/1/0
```

Comment:
Same as above; be sure to use full interface names consistently.

Reply:
Updated Interface Names